react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,19 +1,25 @@
 import React, { Component } from 'react';
+import { array, func } from 'prop-types';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
+
+  renderSquare(i) {
+    return (
+      <Square
+        value={this.props.squares[i]}
+        onClick = {() => this.props.onClick(i)}
```

Comment:
You can pass as a prop the onClick function and 'i' as another prop. Then in the component square, you can create a method class that calls the function with 'i' as a param